### PR TITLE
HTML/CSS & JavaScript capstone project - Conference page

### PR DESCRIPTION
**Pull request by ndikumanaisaie**
- [ ] In this milestone I did the following:

- The description of the project is in this 5 min loom video. 
- [Description of the project] (https://www.loom.com/share/9015ef58eaaf4bfaa85a25942c30dfaf)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,231 +4,178 @@
...
+              </nav>
+            </div>
+            <div class="header-bottom">
+              <nav class="main-top">
+                <a class="cursor-pointer" href="index.html"><img class="site-logo" src="./images/logo1.png" alt="site logo on header "/></a>
+                <ul class="main-head flex items-center">
+                  <li class="subHeaderListStyle"><a href="about.html" class="no-underline gray-text-color">About</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Program</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Join</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Sponsor</a></li>
```
Comment (AnselemOdims) [inline index.html:49]: - [ ] Please kindly ensure to stick to the design guidelines(margins, padding, spacing, font, color etc). In this case, your top header does not have the proper right padding that the original design has. The bottom header does not have the proper padding around it like the original design.

### Your Design
image

### Original Design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        </section>
+      </section>>
+    </div>
+    
+    <footer>
+      <div class="footer">
+        <img class="logo" src="./images/logo1.png" alt="my logo">
+        <p class="main1"> 2022 East African Developper Bujumbura some rights reserved</p>
+      </div>
+    </footer>
```
Comment (AnselemOdims) [inline about.html]: - [ ] Please kindly ensure that the footer for the about page in  the desktop view is dark according to the original design.
- [ ] Please also ensure that the size of the logo in the footer matches the original design

### Your Design
image

### Original Design
image

File: index.html [hunk 3]
```diff
@@ -4,231 +4,178 @@
...
+      </div>
+      <div class="summit-program">
+        <div class="program-card">
+          <div class="card-title">
+            <i class="fa-solid fa-person-chalkboard fa-4x program-img"></i>
+            <p class="title">Speaches</p>
+          </div>
+          <p class="Opening">Listen to the opening speak of software specialists from different East African Countries giving 
+            key notes about IT in their respective countries.
+          </p>
```
Comment (AnselemOdims) [inline index.html]: - [ ] At the moment, there is no proper padding in the program section. Kindly include padding to make it match the original design.
- [ ] Please also ensure that all cards are the same size

### Your Design
image

### Original Design
image

File: index.html [hunk 4]
```diff
@@ -4,231 +4,178 @@
...
-            maxlength="500"
-            required
-          ></textarea>
-          <input aria-label="submit" class="submit" name="submit" type="submit" value="Get In Touch" />
-          <span id="error-msg"></span>
-        </form>
-        <div class="bottom-rule">
-        </div>
+      </div>
+    </section>
```
Comment (AnselemOdims) [inline index.html:154]: - [ ] The tile image is supposed to be positioned slightly above and behind the main image but yours is covering the whole space behind the image. Please kindly ensure the size and positioning matches the original design.
- [ ] In the mobile design, there should be space around speaker cards like that of the original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @ndikumanaisaie , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 4 above)
Comment (zhadier) [reply index.html:154]: - [ ] Great job implementing this change! However it seems to have introduced a bug as the first image is not showing up now. Kindly fix this tiny bug 🌤️ 
image

File: index.html (hunk 1 above)
Comment (zhadier) [reply index.html:49]: - [ ] As requested by the previous reviewer, kindly ensure you add the correct right padding to the top header as you can see there is no padding in the image below

image

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @ndikumanaisaie 👋 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by ndikumanaisaie** [comment]
Hi [zhadier](https://github.com/zhadier) I have corrected the bug and fix the padding problem. I have checked the online version now it is working as I designed it.

**Review by Mre55 — APPROVED** [review]
**Hi @ndikumanaisaie,** 

# Status: APPROVED 🏆 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**